src/onnx/onnx_parser.cpp

```diff
@@ -70,7 +70,7 @@ static literal from_repeated(shape::type_t t, const T& r)
 
 instruction_ref onnx_parser::node_info::make_contiguous(instruction_ref ins) const
 {
-    if(ins->get_shape().standard())
+    if(ins->name() == "contiguous")
```

Comment:
This doesn't seem like this would really fix the issue. Users could still manually build a custom program without a contiguous before reshape, but auto_contiguous could still insert a contiguous that will cause this error. This seems like this should be fixed in auto_contiguous.